### ISSUE TITLE
SVTRv2微调，acc为0,loss变大

### ISSUE THREAD

**Issue by zzt6227** [PaddlePaddle/PaddleOCR#15922]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

SVTRv2微调，acc为0,loss变大，相同数据PPOCR v5微调没有出现问题。

训练日志如下：
[2025/07/01 02:27:36] ppocr INFO: epoch: [17/500], global_step: 58610, lr: 0.000999, acc: 0.947917, norm_edit_dis: 0.990191, CTCLoss: 0.340744, NRTRLoss: 0.879569, loss: 1.223313, avg_reader_cost: 0.00314 s, avg_batch_cost: 0.61449 s, avg_samples: 131.2, ips: 213.51136 samples/s, eta: 12 days, 3:21:01, max_mem_reserved: 34473 MB, max_mem_allocated: 26919 MB
[2025/07/01 02:27:42] ppocr INFO: epoch: [17/500], global_step: 58620, lr: 0.000999, acc: 0.927083, norm_edit_dis: 0.985860, CTCLoss: 0.655599, NRTRLoss: 0.899152, loss: 1.558683, avg_reader_cost: 0.00348 s, avg_batch_cost: 0.62337 s, avg_samples: 128.0, ips: 205.33459 samples/s, eta: 12 days, 3:20:58, max_mem_reserved: 34473 MB, max_mem_allocated: 26919 MB
[2025/07/01 02:27:49] ppocr INFO: epoch: [17/500], global_step: 58630, lr: 0.000999, acc: 0.932292, norm_edit_dis: 0.985160, CTCLoss: 0.616683, NRTRLoss: 0.898629, loss: 1.508727, avg_reader_cost: 0.00298 s, avg_batch_cost: 0.63258 s, avg_samples: 115.2, ips: 182.11062 samples/s, eta: 12 days, 3:20:58, max_mem_reserved: 34473 MB, max_mem_allocated: 26919 MB
[2025/07/01 02:27:55] ppocr INFO: epoch: [17/500], global_step: 58640, lr: 0.000999, acc: 0.921875, norm_edit_dis: 0.983845, CTCLoss: 0.558971, NRTRLoss: 0.898700, loss: 1.470766, avg_reader_cost: 0.00355 s, avg_batch_cost: 0.61767 s, avg_samples: 137.6, ips: 222.77215 samples/s, eta: 12 days, 3:20:54, max_mem_reserved: 34473 MB, max_mem_allocated: 26919 MB
[2025/07/01 02:28:01] ppocr INFO: epoch: [17/500], global_step: 58650, lr: 0.000999, acc: 0.500000, norm_edit_dis: 0.794610, CTCLoss: 10.862615, NRTRLoss: 2.009816, loss: 12.975531, avg_reader_cost: 0.00252 s, avg_batch_cost: 0.61832 s, avg_samples: 134.4, ips: 217.36451 samples/s, eta: 12 days, 3:20:50, max_mem_reserved: 34473 MB, max_mem_allocated: 26919 MB
[2025/07/01 02:28:07] ppocr INFO: epoch: [17/500], global_step: 58660, lr: 0.000999, acc: 0.005208, norm_edit_dis: 0.039146, CTCLoss: 47.741196, NRTRLoss: 3.093670, loss: 51.088236, avg_reader_cost: 0.00300 s, avg_batch_cost: 0.60830 s, avg_samples: 147.2, ips: 241.98523 samples/s, eta: 12 days, 3:20:44, max_mem_reserved: 34473 MB, max_mem_allocated: 26919 MB
[2025/07/01 02:28:13] ppocr INFO: epoch: [17/500], global_step: 58670, lr: 0.000999, acc: 0.000000, norm_edit_dis: 0.005136, CTCLoss: 41.563820, NRTRLoss: 2.936639, loss: 44.435169, avg_reader_cost: 0.00281 s, avg_batch_cost: 0.60441 s, avg_samples: 140.8, ips: 232.95397 samples/s, eta: 12 days, 3:20:36, max_mem_reserved: 34473 MB, max_mem_allocated: 26919 MB
[2025/07/01 02:28:19] ppocr INFO: epoch: [17/500], global_step: 58680, lr: 0.000999, acc: 0.000000, norm_edit_dis: 0.002060, CTCLoss: 38.339428, NRTRLoss: 2.765859, loss: 41.176798, avg_reader_cost: 0.00346 s, avg_batch_cost: 0.61302 s, avg_samples: 140.8, ips: 229.68174 samples/s, eta: 12 days, 3:20:30, max_mem_reserved: 34473 MB, max_mem_allocated: 26919 MB

### 🏃‍♂️ Environment (运行环境)

paddleocr                3.0.2
paddlepaddle-gpu         3.0.0

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

训练命令：python -m paddle.distributed.launch --log_dir=exp18/logs --gpus 0,1,2,3 ./tools/train.py -c ./config/exp18/config.yml
配置文件：
Global:
  debug: false
  use_gpu: true
  epoch_num: 500
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./results/exp18
  save_epoch_step: 50
  eval_batch_step: [10000, 2000]
  cal_metric_during_train: True
  pretrained_model: ./model/paddle/openatom_rec_svtrv2_ch_train/best_accuracy.pdparams
  checkpoints: 
  save_inference_dir:
  use_visualdl: True
  infer_img: ./data/train/03/val/imgs
  character_dict_path: ./data/train/03/latin_dict_global.txt
  max_text_length: &max_text_length 50
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./data/train/03/predicts_svrtv2.txt
  d2s_train_image_shape: [3, 48, 320]

Optimizer:
  name: AdamW
  beta1: 0.9
  beta2: 0.999
  epsilon: 1.e-8
  weight_decay: 0.05
  no_weight_decay_name: norm
  one_dim_param_no_weight_decay: True
  lr:
    name: Cosine
    learning_rate: 0.001 # 8gpus 192bs
    warmup_epoch: 5

Architecture:
  model_type: rec
  algorithm: SVTR_HGNet
  Transform:
  Backbone:
    name: SVTRv2
    use_pos_embed: False
    dims: [128, 256, 384]
    depths: [6, 6, 6]
    num_heads: [4, 8, 12]
    mixer: [['Conv','Conv','Conv','Conv','Conv','Conv'],['Conv','Conv','Global','Global','Global','Global'],['Global','Global','Global','Global','Global','Global']]
    local_k: [[5, 5], [5, 5], [-1, -1]]
    sub_k: [[2, 1], [2, 1], [-1, -1]]
    last_stage: False
    use_pool: True
  Head:
    name: MultiHead
    head_list:
      - CTCHead:
          Neck:
            name: svtr
            dims: 256
            depth: 2
            hidden_dims: 256
            kernel_size: [1, 3]
            use_guide: True
          Head:
            fc_decay: 0.00001
      - NRTRHead:
          nrtr_dim: 384
          max_text_length: *max_text_length
          num_decoder_layers: 2

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
    - NRTRLoss:

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: MultiScaleDataSet
    ds_width: false
    data_dir: ./data/train/03/train/imgs
    ext_op_transform_idx: 1
    label_file_list:
    - ./data/train/03/train/ppocr_label_rec_train.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecAug:
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
   sampler:
    name: MultiScaleSampler
    scales: [[320, 32], [320, 48], [320, 64]]
    first_bs: &bs 192
    fix_bs: false
    divided_factor: [8, 16] # w, h
    is_training: True
  loader:
    shuffle: true
    batch_size_per_card: *bs
    drop_last: true
    num_workers: 8
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./data/train/03/val/imgs
    label_file_list:
    - ./data/train/03/val/ppocr_label_rec_val.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 128
    num_workers: 4

**Comment by zhangyubo0722** [comment]
可以尝试调小轮数以及调小学习率

**Comment by zzt6227** [comment]
> 可以尝试调小轮数以及调小学习率

到第17个epoch就开始崩了，所以应该不是轮数的问题；学习率0.001是正常的，关键是ppocrv5的识别模型相同的学习率就正常训练啊，所以可能也不是学习率问题

**Comment by zhangyubo0722** [comment]
两个模型不一样，用的学习率也不一样，且强依赖于训练数据，所以可以尝试一下

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
